src/reduxStore/selectors/signedMessagesSelectors.ts

```diff
@@ -0,0 +1,4 @@
+import { RootState } from '../store';
+
+export const signedMessagesSliceSelector = (state: RootState) =>
+  state.signedMessages;
```

Comment:
maybe signMessageInfo? since there is never more than 1 signed message at a time